dtbootstrap-anima-keyinfra.xml

```diff
@@ -1489,16 +1489,40 @@ locator3  = [O_IPv6_LOCATOR, fe80::1234, 41, nil]]]></artwork>
         <t>This is done with an HTTP GET using the operation path value of
         "/requestauditlog".</t>
 
-        <t>The Registrar MUST HTTP POSTs the same Registrar voucher-request as it did when requesting a
-        Voucher. It is posted to the /requestauditlog URI instead. The
-        "idevid-issuer" and "serial-number" informs the MASA
-        server which log is requested so the appropriate log can be prepared
-        for the response. Using the same media type and message minimizes
-        cryptographic and message operations although it results in additional
-        network traffic. The relying MASA server implementation MAY leverage
-        internal state to associate this request with the original, and by now
-        already validated, Registrar voucher-request so as to avoid an extra crypto
-        validation.</t>
+        <t>
+          The Registrar MUST HTTP POSTs the same Registrar voucher-request
+          as it did when requesting a
+          Voucher. It is posted to the /requestauditlog URI instead.
+          The "idevid-issuer" and "serial-number" informs the MASA
+          server which log is requested so the appropriate log can be prepared
+          for the response.
+          Using the same media type and message minimizes
+          cryptographic and message operations although it results in additional
+          network traffic.
+          The relying MASA server implementation MAY leverage internal state
+          to associate this request with the original, and by now already
+          validated, Registrar voucher-request so as to avoid an extra crypto
+          validation.
+        </t>
+        <t>
+          Rather than returning the audit log as a response to the POST (with
+          a return code 200), the MASA MAY instead return a 201 ("Created")
+          RESTful response (<xref target="RFC7231" /> section 7.1) containing
+          a URL to the prepared (and easily cachable) audit response.
+        </t>
+        <t>
+          MASA servers that return URLs SHOULD take care to make the returned
+          URL unguessable.  URLs containing a database number
+          such as https://example.com/auditlog/1234 or the EUI of the device
+          such https://example.com/auditlog/10-00-00-11-22-33, would be
+          easily enumerable by an attacker.  It is recommended put to put
```

Comment:
This assumes the attacker is unauthenticated? But it is an authenticated connection.

I had thought we had language that the Registrar had to be in the log (e.g. had to claim the device) before the audit log would be sent to them. That isn't in the language of the draft though. Hmm. I don't have a strong opinion on which approach is best but note that the protections are easier with that method because they all come around to Registrar-MASA enhancements around sales channel improvements.